.rubocop_todo.yml

```diff
@@ -159,12 +158,6 @@ Metrics/AbcSize:
     - 'lib/deimos/utils/schema_controller_mixin.rb'
```

Comment:
inline_consumer and schema_controller_mixin can be removed as the files are removed in the PR.
```
    - 'lib/deimos/utils/inline_consumer.rb'
    - 'lib/deimos/utils/schema_controller_mixin.rb'
```